integrations/refined/src/main/scala/sttp/tapir/codec/refined/TapirCodecRefined.scala

```diff
@@ -36,7 +36,12 @@ trait TapirCodecRefined extends LowPriorityValidatorForPredicate {
       }(_.value)
   }
 
-  //
+  implicit def uuidTapirSchema(implicit
+      vSchema: Schema[String],
+      refinedValidator: Validate[String, Uuid],
+      refinedValidatorTranslation: ValidatorForPredicate[String, Uuid]
+  ): Schema[String Refined Uuid] =
+    refinedTapirSchema[String, Uuid].format("uuid")
```

Comment:
Shouldn't we add this to `Schema.schemaForUUID` ?

Reply:
Good call, added